views/endMeeting.ejs

```diff
@@ -48,16 +48,16 @@
              Check out the GitHub repo.
             </a> </br></br>
               Have a killer idea for how we can make this app even better?
-            <a  href="https://tokbox.com/account/user/signup?icid=startbuidlingforfree_freeconferencing_tokboxfreetrialsignup_btn" target="_blank" rel="noopener"> Sign up for a Vonage Video API account</a> and start hacking today!
+            <a  href="https://tokbox.com/account/user/signup<%= startBuilidingIcidQueryString %>" target="_blank" rel="noopener"> Sign up for a Vonage Video API account</a> and start hacking today!
```

Comment:
Should we keep the "?" in the code or force that to me in the config var?

Reply:
Im keeping the "?" in the code, but not in the view template. im including it only if the config is set. See this for a better understanding:
https://github.com/opentok/opentok-rtc/blob/ECO-5915/server/serverMethods.js#L218
this is to avoid to show a question mark with nothing after it if the setting is unset

Reply:
Basically in the config for this particular  case you just need to set it to `startbuidlingforfree_freeconferencing_tokboxfreetrialsignup_btn` the "?icid=" gets auto included if the setting is found

Reply:
Brilliant.